.gitignore

```diff
@@ -1,3 +1,7 @@
 instrumented/
 tests/coverage.html
 docs/_build
+
+/.idea/
+
+*.iml
```

Comment:
Please remove these directories, you should add them to you local global .gitignore